.github/workflows/build-pdf.yml

```diff
@@ -0,0 +1,33 @@
+name: Build LaTeX document
+on: [push, pull_request]
+jobs:
+  build-pdf:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v2
+      - name: Compile LaTeX document
+        uses: xu-cheng/latex-action@v2
+        with:
+          root_file: hydras.tex
+          working_directory: doc
+          latexmk_use_lualatex: true
+      - name: Extract PDF
+        run: |
+          mkdir public
```

Comment:
shouldn't this be `mkdir -p` if you want to avoid errors when the directory exists?

Reply:
Yeah, why not make it future-proof indeed!

Reply:
```suggestion
          mkdir -p public
```